File: src/main/java/com/kaiwen/Hello.java
package com.kaiwen;

public class Hello {
    public static void main(String[] args) {
//        System.out.println("Hello World");
//        new Person().hello();
        Person p=new Person();
        p.hello();
        //基本資料型態(小寫)
        int age = 19; //32 bits
        //Wrapper class包裝類別
        Integer age2 = 19;
        //Method
        age2.byteValue();

        short s = 1;//16 bits
        long l = 2 ;//64 bits
        char c ='A'; //16 bits
        Character c2 = 'A';
        byte b =  120 ;//8 bits
        float weight = 66.5f; //32 bits
        double height = 1.7; //64 bits
        boolean adult = true;
        boolean enroll = false;
        //Reference data type參照資料型態(大寫)
        String name = "Tom";


    }

}

File: src/main/kotlin/com/kaiwen/Hello.kt
package com.kaiwen

fun main() {
//    println("Hello Kotlin");
//    Human().hello();
    val h =Human();
    h.hello();

    //自動推斷變數型態
    val age = 19;
    //age = 20; ##Val cnanot be reassigned
    //val age : Int = 19;
    var weight = 66.5; //Double
    var weight2 = 66.5f;
    var weight3 : Float = 66.5f;
    var name : String;
    name = "Kaiwen";

    Int
    Short
    Long
    //val (value) 給值後不可變
    //var (variable) 可變的
}

class Human{
    fun hello(){
        println("Hello Kotlin");
    }
}